src/kirby/components/segmented-control/segmented-control.component.tns.html

```diff
@@ -0,0 +1,15 @@
+  <StackLayout orientation="horizontal" horizontalAlignment="left" class="segment-control">
+    <ng-container *ngFor="let item of segmentItems">
```

Comment:
Consider moving `*ngFor="let item of segmentItems"` to `GridLayout`-element (below)